core/src/main/scala/org/apache/spark/package.scala

```diff
@@ -41,7 +41,53 @@ package org.apache
  * level interfaces. These are subject to changes or removal in minor releases.
  */
 
+import java.util.Properties
+
 package object spark {
-  // For package docs only
-  val SPARK_VERSION = "2.0.0-SNAPSHOT"
+
+  private object SparkBuildInfo {
+    val resourceStream = Thread.currentThread().getContextClassLoader.
```

Comment:
Might also be a good idea to explicitly check that  `resourceStream` is not null (instead of throwing an NPE). Shouldn't ever happen, but well.